internal/build/lifecycle_execution.go

```diff
@@ -35,6 +38,7 @@ type LifecycleExecution struct {
 	os           string
 	mountPaths   mountPaths
 	opts         LifecycleOptions
+	WorkingDir   string
```

Comment:
I don't love making this public just for tests. But the alternatives were a bit more complicated. If anyone has feedback here, I'd be glad to hear it.